Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1042,6 +1042,16 @@ public long GetNextTxNonce(Address address)
 
             _logger.Debug("Trying to append block {blockIndex}: {block}", block?.Index, block);
 
+            if (block.Transactions.Count() is { } txCount &&
```

Comment:
It is a question for this line, `block.Transactions.Count()`'s return value is nullable? I wonder how this line works.

Reply:
It does not have to necessarily use pattern matching here; it just functions as an inline variable declaration (`txCount`)…

Reply:
I understood. Thanks for your kind description. 🙏